business/definitionService.js

```diff
@@ -500,6 +504,14 @@ class DefinitionService {
     }
   }
 
+  async _ensureUrls(coordinates, definition) {
+    const homePageUrl = buildHomepageUrl(coordinates)
+    const componentUrl = buildComponentUrl(coordinates)
```

Comment:
see https://github.com/clearlydefined/service/issues/403#issuecomment-465585617 from @pombredanne 

I agree, we should come up with a better naming here


here's an updated proposal

```json
{
  "described": {
      "urls": {
           "registry": "https://www.npmjs.com/package/tiny-attribution-generator",
           "version": "https://www.npmjs.com/package/tiny-attribution-generator/v/0.0.6",
           "download": "https://registry.npmjs.com/tiny-attribution-generator/tiny-attribution-generator-0.0.6.tgz"
       }
  }
}
```

```json
{
  "described": {
      "urls": {
           "registry": "https://github.com/clearlydefined/service",
           "version": "https://github.com/clearlydefined/service/tree/9f6f79a17460c92fb3876d9f485411d43aa70374",
           "download": "https://github.com/clearlydefined/service/archive/9f6f79a17460c92fb3876d9f485411d43aa70374.zip"
       }
  }
}
```

Reply:
Makes sense for me, i'm going to update this